docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,6 +6,79 @@
 
 :::
 
+## v1.9.4 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.4))
+
+This version is backwards compatible to [v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0). It is optional, but encouraged. The supported plugin version is `20`.
+
+**This version modifies the db format. The db format is compatible with v1.9.3, but not v1.9.2 or earlier. After running a node with v1.9.4 attempting to run a node with a version earlier than v1.9.3 may report a fatal error on startup.**
+
+### PeerList Gossip Optimization
```

Comment:
```suggestion
**PeerList Gossip Optimization**
```
Please replace all of them with ** **, otherwise, this will generate unnecessary section headers as in

<img width="1531" alt="Screen Shot 2022-12-07 at 1 57 56 PM" src="https://user-images.githubusercontent.com/12171681/206294524-0285cf00-f7ba-42bf-b38f-2199323d3963.png">